src/back/spv/writer.rs

```diff
@@ -1320,6 +1355,234 @@ impl Writer {
         })
     }
 
+    /// Emit code to compute the length of a run-time array.
+    ///
+    /// Given `array`, an expression referring to the final member of a struct,
+    /// where the member in question is a runtime-sized array, return the insn
```

Comment:
what does "insn" stand for?

Reply:
"Instruction". I think it's a GCC-ism. I'll write it out.